milton-processor/scribe.ts

```diff
@@ -86,9 +88,16 @@ app.post("/simplify", async (req, res) => {
           updatedAt: Date.now(),
         });
         cache.saveManuscript(url, manuscript);
+
+        // try saving a PDF
+        console.log(`Caching snapshot version of ${url}`);
+        // const pdfData = await snapshot.fetchPDF(url);
+        // cache.saveRawContents(url, cache.PDF_PATH, pdfData, 'application/pdf');
```

Comment:
Commented out code here, just a heads up if you didn't intend to commit it

Reply:
removed